__tests__/randomUUID.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const assert = require('assert');
+
+const randomUUID = require('../randomUUID');
+
+
+function testMatch(uuid) {
+  assert.match(
+    uuid,
+    /^[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/);
+}
+
+
+// Generate a number of UUID's to make sure we're not just generating the same
+// value over and over and to make sure the batching changes the random bytes.
+test("Generate multiple UUIDs", function()
+{
+  const last = new Set([
+    '00000000-0000-0000-0000-000000000000'
+  ]);
+
+  for (let n = 0; n < 130; n++) {
+    const uuid = randomUUID();
+    assert(!last.has(uuid));
+    last.add(uuid);
+    assert.strictEqual(typeof uuid, 'string');
+    assert.strictEqual(uuid.length, 36);
```

Comment:
Redundant with `testMatch()` (which will fail if either of these fails, right?) 
```suggestion
```

Reply:
Yes, they would fail. The same, tests extracted from Node.js code.